WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift

```diff
@@ -19,6 +21,17 @@ class SiteDesignPreviewViewController: TemplatePreviewViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationItem.leftBarButtonItem = CollapsableHeaderViewController.closeButton(target: self, action: #selector(closeButtonTapped))
+        setPrimaryActionButtonTitle()
+    }
+
+    private func setPrimaryActionButtonTitle() {
+        primaryActionButton.setTitle(createsSite ? TextContent.createSiteButton : TextContent.chooseButton, for: .normal)
+    }
+
+    private enum TextContent {
+        static let previewTitle = NSLocalizedString("Preview", comment: "Title for screen to preview a selected homepage design.")
+        static let createSiteButton = NSLocalizedString("Create Site", comment: "Title for the button to progress with creating the site with the selected design.")
+        static let chooseButton = NSLocalizedString("Choose", comment: "Title for the button to progress with the selected site homepage design.")
```

Comment:
Note that since those 2 new strings were not present at the time you did the code freeze and got added in `release/*` later, those new strings sadly won't be sent for translation and won't be translated in time for 19.7 😞

Reply:
👋 @AliSoftware thanks for all your information recently about translations. Could I get one bit clarified?

If I understand correctly, "Create Site" and "Choose" are keys* for the translated strings. These keys existed prior to this change in WPiOS and had the same meaning [[one](https://github.com/wordpress-mobile/WordPress-iOS/blob/7ea0fb21958cbd1516e236b99b1b28322b776f67/WordPress/Resources/en.lproj/Localizable.strings#L2121)] [[two](https://github.com/wordpress-mobile/WordPress-iOS/blob/7ea0fb21958cbd1516e236b99b1b28322b776f67/WordPress/Resources/en.lproj/Localizable.strings#L1462)].

Does this still mean the translations won't work for this view? I ran it locally and set it to a random language (German) and saw it was translated. Thanks!

*I realize we may be getting away from the English string being the key soon.

Reply:
Ah! I might have missed something when looking at the diff then, as I was convinced that those were newly introduced strings — as opposed to already existing ones that we just moved around or reused in a different context.

If they were already part of the strings that got extracted in the generated `Localizable.strings` back when Gio did the code freeze (maybe because they were already used / present in another part of the code) then we should be all good for those then!

Reply:
Cool! 

"Create Site" is a "new" string to that view, but it was used (in the same context) in WPiOS prior to this change, so my hope is that the translation for that key is just reused.

"Choose" already existed on this view, it was only moved. 👍